pkg/lvm/lvm_util.go

```diff
@@ -511,9 +606,25 @@ func parseVolumeGroup(m map[string]string) (apis.VolumeGroup, error) {
 		quantity := resource.NewQuantity(sizeBytes, resource.BinarySI)
 		*value = *quantity //
 	}
+
+	vg.Permission = getIntFieldValue("vg_permissions", m["vg_permissions"])
```

Comment:
Why do we have to convert from int to string? Is it because, we get the output as int ?

Reply:
Or, is it the other way round here?

Reply:
To get this in the form of a metric which can be plotted in grafana dashboard.